File: Backend/src/main/java/com/tarikturkdil/dto/BoardResponse.java
package com.tarikturkdil.dto;

import lombok.Data;

@Data
public class BoardResponse extends DtoBase{

	private String name;
    private String description;
    private boolean isSecret;
    // İleride panonun sahibini de dönmek istersen UserResponse ekleyebilirsin
}

File: Frontend/app/src/main/java/com/tarikturkdil/photoProject/data/remote/dto/BoardResponse.kt
package com.tarikturkdil.photoProject.data.remote.dto

import kotlinx.serialization.Serializable

@Serializable
data class BoardResponse(
    val id: Long,
    val name: String,
    val description: String? = null,
    val secret: Boolean = false,
    val createTime: String? = null
)